packages/nuxt/test/import-protection.test.ts

```diff
@@ -14,6 +14,9 @@ const testsToTriggerOn = [
   ['/root/node_modules/@vue/composition-api', 'components/Component.vue', true],
   ['@vue/composition-api', 'components/Component.vue', true],
   ['@nuxt/kit', 'components/Component.vue', true],
+  ['nuxt/config', 'components/Component.vue', true],
+  ['nuxt/kit', 'components/Component.vue', true],
+  ['nuxt/schema', 'components/Component.vue', true],
```

Comment:
Shall we include `nuxt` itself to the list or will it be conflicting with subpaths?